src/pages/sweepstake/mintform/index.jsx

```diff
@@ -0,0 +1,288 @@
+  const [errorMessage, setErrorMessage] = useState()
+
+  function fileHandleChange(event) {
+
+    // Verify if the user cancel the file
+    if (event.target.files[0] == undefined || event.target.files[0] == null) return;
+
+    const viewFileTypes = [
+      "image/jpeg",
+      "image/png",
+      "image/gif",
+      "image/svg+xml",
+      "video/mp4",
+      "video/ogg",
+      "audio/mp3",
+      "audio/mpeg"
+    ];
+
+
+    if (event.target.value.includes(viewFileTypes[0] || viewFileTypes[1] || viewFileTypes[2] || viewFileTypes[3] || viewFileTypes[4] || viewFileTypes[5])) {
+      return;
+    }
+
+    if (event.target.files[0].size / (1024 * 1024) > 50) {
+      setErrorMessage("File cannot be greater than 50MB")
+      return
+    }
+
+    if (["image/jpeg", "image/png", "image/gif", "image/svg+xml"].includes(event?.target.files[0].type)) {
+      setUrlPreviewFile(URL.createObjectURL(event.target.files[0]));
+    }
+
+    if (["audio/mp3", "audio/mpeg", "video/mp4", "video/ogg"].includes(event?.target.files[0].type)) {
+      setMediaSrc(URL.createObjectURL(event.target.files[0]))
+    }
+
+    // Save file states
+    setFile(event.target.files[0]);
+  }
+
+  const processValues = async () => {
+
+    const MINTER = await contractERC721EsolidarSweepstake.methods.hasRole(process.env.NEXT_PUBLIC_MINTER_ROLE, address).call()
+    if(MINTER == false){
+      setErrorMessage("You must be a charity to perform this action")
+      return
+    }
+
+    await performActions(async (kit) => {
+      let totalBalance = await kit.getTotalBalance(address)
+      if(!(totalBalance > 0)){
+        setErrorMessage("You don't have enough CELO to mint")
+        return
+      }
+    })
+
+
+
+
+    setErrorMessage('')
+
+    try {
+      const attributes = [
+        { trait_type: 'prop1', value: propOne },
+        { trait_type: 'prop2', value: propTwo },
+        { trait_type: 'prop3', value: propThree },
+      ]
+
+      // Send file to IPFS
+      const formDataFile = new FormData()
+      formDataFile.append("file", file)
+
+      var formData = new FormData();
+      formData.append("file", file);
+
+      try {
+        const resp = await axios.post('/api/ipfs', formData, {
+          headers: {
+            'Content-Type': 'multipart/form-data'
+          }
+        })
+
+        var respFile = resp.data
+      } catch (e) {
+        console.log(e)
+      }
+
+      // Generate metadata json
+      const metadataJson = {
+        'image': `ipfs://${respFile.ipfs}`,
+        'external_url': process.env.NEXT_PUBLIC_S3_URL + respFile.filename,
+        'name': title,
+        'description': description,
+        'attributes': attributes
+      }
+
+      formData = new FormData()
+      formData.append("metadata", JSON.stringify(metadataJson))
+
+      // Mint tokenURI respMetadado
+      await performActions(async (kit) => {
+        const resp = await axios.post('/api/ipfs', formData)
+        const tokenUri = resp.data.tokenUri
```

Comment:
Codacy has a fix for the issue: [Use object destructuring.](https://app.codacy.com/gh/esolidar/web3/pullRequest?prid=9542591)
```suggestion
        const {tokenUri} = resp.data
```